sdcm/cluster.py

```diff
@@ -3271,13 +3271,46 @@ def download_scylla_manager_log(self):
                 self.log.error('Error downloading scylla manager log : %s',
                                str(details))
 
+    def collect_scylla_monitoring_logs(self, node):
+        log_paths = []
+        monitoring_dockers = self._get_running_monitoring_dockers(node)
+        for m_docker in monitoring_dockers:
+            m_docker_log_path = self._collect_monitoring_docker_log(node, m_docker)
+            if m_docker_log_path:
+                log_paths.append(m_docker_log_path)
+
+        return log_paths
+
+    def _get_running_monitoring_dockers(self, node):
+        monitoring_dockers = []
+        result = node.remoter.run("sudo docker ps --format {{.Names}}", ignore_status=True)
```

Comment:
docker ps **-a** since when container is down it won't be running and this is an interesting case for us to investigate :)

Reply:
done